src/Sentry/buildTransitive/Sentry.targets

```diff
@@ -9,6 +9,12 @@
     <SentryAttributesFile>Sentry.Attributes$(MSBuildProjectExtension.Replace('proj', ''))</SentryAttributesFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(PkgSentry)' == ''">
+    <!-- If Sentry was added via PackageReference, PkgSentry will already be set to the path of the Sentry nuget package.
+         However, older-style projects using packages.config will need us to resolve the path manually. -->
```

Comment:
`packages.config` is this a thing on .NET Core apps? TIL

Reply:
Ah, right you are... it's a netframework thing only.